lib/genome/online_updater.rb

```diff
@@ -73,6 +73,16 @@ def create_interaction_claim_publication(interaction_claim, pmid)
       interaction_claim.publications << publication unless interaction_claim.publications.include? publication
     end
 
+    def create_interaction_claim_publication_by_pmcid(interaction_claim, pmcid)
+      uri = URI.parse("https://www.ncbi.nlm.nih.gov/pmc/utils/idconv/v1.0/?ids=#{pmcid}&format=json&tool=DGIdb&email=help@dgidb.org")
+      res = Net::HTTP.get_response(uri)
```

Comment:
Should this be using the `make_get_request` helper that you refactored in this PR?

Reply:
Done.